src/game.cpp

```diff
@@ -6842,6 +6865,30 @@ Creature const* game::critter_at( const tripoint &p, bool allow_hallucination )
     return const_cast<game*>(this)->critter_at( p, allow_hallucination );
 }
 
+Creature *game::critter_with_id( long id )
+{
+    if( id == 1 ) {
```

Comment:
Having several player character in one world is possible and support. The player character may not always have id 1. It is already assigned a unique ID via `player::setID`.